README.md

```diff
@@ -99,6 +101,7 @@ Customizable variables can be created with the following properties:
 |max|The maximum value generated|
 |values|Defines an array of possible values. Example ["on", "off"]|
 |customlengthstring|Creates a random string of n bytes. Provide n as parameter|
+|sequence|Create a sequence of values as defined in `values` property, producing each one. The values itself can reference other variables|

```

Comment:
Would "producing one after the other" be a bit clearer than "producing each one"?